msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/local/TestCase99656.java

```diff
@@ -103,6 +108,12 @@ public void handleUserInteraction() {
         silentLatch.await(TokenRequestTimeout.SILENT);
 
         // second interactive request
+        // wait about a minute here to throttle usage of AUTO MFA account
+        ThreadUtils.sleepSafely(
+                (int) TimeUnit.MINUTES.toMillis(1),
+                TAG,
+                "Problem occurred while sleeping safely to throttle AUTO MFA requests."
```

Comment:
I mean, there's nothing really safe about it, it just takes care of the boilerplate for you in terms of thread.sleep.

Reply:
Yeah...I couldn't figure out a better message to log here :D